src/modes/thread_index/thread_index.cc

```diff
@@ -32,10 +32,10 @@ namespace Astroid {
     list_store = Glib::RefPtr<ThreadIndexListStore>(new ThreadIndexListStore ());
     queryloader.list_store = list_store;
 
-    list_view  = Gtk::manage(new ThreadIndexListView (this, list_store));
```

Comment:
Same here. All these are subclassed from `Mode` which is a `Gtk::Box` or similar.

Reply:
But none of these are added to another GTK object, or did I miss something?

Reply:
It's added to the scroll window, which is added through the add_pane call. Might still be cleaned up though.. But all Gtk Widgets should either be used or it should be removed.

Reply:
Ok, should be reverted now.